docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/ecommerce-tracking/index.md

```diff
@@ -0,0 +1,702 @@
+
+```kotlin
+val product = Product("productId", "clothes/shirts", "EUR", 100.50)
+val event = AddToCart(listOf(product), 200, "EUR")
+
+tracker.track(event)
+```
+
+  </TabItem>
+  <TabItem value="android-java" label="Android (Java)">
+
+```java
+Product product = new Product("productId", "clothes/shirts", "EUR", 100.50);
+List<Product> products = new ArrayList<>();
+products.add(product);
+AddToCart event = new AddToCart(products, 200, "EUR");
+
+tracker.track(event);
+```
+
+  </TabItem>
+</Tabs>
+
+### RemoveFromCart
+
+Tracking one or more products being removed from a cart.
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="ios" label="iOS" default>
+
+Coming soon!
+
+  </TabItem>
+  <TabItem value="android" label="Android (Kotlin)">
+
+```kotlin
+val product = Product("productId", "clothes/shirts", "EUR", 100.50)
+val event = RemoveFromCart(listOf(product), 200, "EUR")
+
+tracker.track(event)
+```
+  </TabItem>
+  <TabItem value="android-java" label="Android (Java)">
+
+```java
+Product product = new Product("productId", "clothes/shirts", "EUR", 100.50);
+List<Product> products = new ArrayList<>();
+products.add(product);
+RemoveFromCart event = new RemoveFromCart(products, 200, "EUR");
+
+tracker.track(event);
+```
+  </TabItem>
+</Tabs>
+
+### ProductListView
+
+Track an impression of a list of products. You can optionally provide a name for the list.
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="ios" label="iOS" default>
+
+Coming soon!
+
+  </TabItem>
+  <TabItem value="android" label="Android (Kotlin)">
+
+```kotlin
+val product = Product("productId", "software", "USD", 99.99)
+val event = ProductListView(listOf(product), "snowplowProducts")
+
+tracker.track(event)
+```
+  </TabItem>
+  <TabItem value="android-java" label="Android (Java)">
+
+```java
+Product product = new Product("productId", "software", "USD", 99.99);
+List<Product> products = new ArrayList<>();
+products.add(product);
+ProductListView event = new ProductListView(products, "snowplowProducts");
+
+tracker.track(event);
+```
+  </TabItem>
+</Tabs>
+
+### ProductListClick
+
+Track a specific product being selected from a list. You can optionally provide a name for the list.
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="ios" label="iOS" default>
+
+Coming soon!
+
+  </TabItem>
+  <TabItem value="android" label="Android (Kotlin)">
+
+```kotlin
+val product = Product("productId", "software", "USD", 99.99)
+val event = ProductListClick(product, "snowplowProducts")
+
+tracker.track(event)
+```
+  </TabItem>
+  <TabItem value="android-java" label="Android (Java)">
+
+```java
+Product product = new Product("productId", "software", "USD", 99.99);
+ProductListClick event = new ProductListClick(product, "snowplowProducts");
+
+tracker.track(event);
+```
+  </TabItem>
+</Tabs>
+
+### CheckoutStep
+
+Track the completion of a step in the checkout funnel, along with common checkout properties.
```

Comment:
Might be worth noting what are the checkout properties? Alternatively we could also link to the API docs for the full list of properties to avoid having to duplicate it here.